APCAppCore/APCAppCore/UI/TabBarControllers/Profile/APCProfileViewController.m

```diff
@@ -1183,23 +1165,18 @@ - (void) imagePickerControllerDidCancel:(UIImagePickerController *)picker
 
 - (void)taskViewController:(ORKTaskViewController *) __unused taskViewController didFinishWithReason:(ORKTaskViewControllerFinishReason)reason error:(nullable NSError *)error
 {
-    if (reason == ORKTaskViewControllerFinishReasonCompleted)
-    {
+    if (reason == ORKTaskViewControllerFinishReasonCompleted) {
```

Comment:
Okay. I usually use the following but am fine with moving the else on the same line as the closing bracket of the if.

```
if (a) {
    b;
}
else {
    c;
}
```